decode/clarity.ts

```diff
@@ -52,7 +52,11 @@ export function decode(input: string): DecodedPayload {
                 break;
             case Event.Metric:
                 if (payload.metric === undefined) { payload.metric = []; }
-                payload.metric.push(data.decode(entry) as MetricEvent);
+                let metric = data.decode(entry) as MetricEvent;
```

Comment:
Could you change the type of the key from a string to an int. Minor but it adds one more unnecessary change to the back end (the metric id is a integer).

Reply:
It is an integer. I think you may receive it as a string because by default JSON.stringify (native function) treats keys of an object as string. There's not much I can do without overloading JSON.stringify, which I would want to avoid to keep things as native as possible. Nothing should change for you though from previous version - it's still the same schema.